packages/boot/test/bootstrapTests/test-orchestration.ts

```diff
@@ -215,3 +339,5 @@ test.serial('stakeAtom - smart wallet', async t => {
     'delegate fails with invalid validator',
   );
 });
+
+test.todo('deposit to LCA fails, payment should be returned');
```

Comment:
Currently only simulating failed LCA -> ICA transfer. This captures the last nested block of the `Deposit()` `try / catch`, but we have nothing testing the first `depositToSeat`, which depends on  `await Object.values(payments)[0]`. Suggestions on how to simulate this welcome.